src/pages/quickstarts/native/expo.mdx

```diff
@@ -33,21 +33,42 @@ Once you have a Expo application ready, you need to install Clerk's Expo SDK.
   </Tab>
 </Tabs>
 
-## Mount `<ClerkProvider>`
+<Callout type="warning">
```

Comment:
This isn't required to work with Clerk, this is all you need to work with Clerk and OAuth so we can remove this

```
"dependencies": {
    "@clerk/clerk-expo": "^0.17.2",
    "expo": "~48.0.15",
    "expo-status-bar": "~1.4.4",
    "react": "18.2.0",
    "react-native": "0.71.7"
  },
  "devDependencies": {
    "@babel/core": "^7.20.0"
  },  ```